src/scaffold/entry_type.rs

```diff
@@ -101,6 +101,15 @@ pub fn scaffold_entry_type(
         }
     };
 
+    let widget_fields = fields
+        .iter()
+        .map(|f| f.clone())
+        .filter(|f| match f.widget {
+            Some(_) => true,
+            None => false
+        })
+        .collect();
```

Comment:
A little more cleaner 🙂

```suggestion
    let widget_fields = fields
        .iter()
        .cloned()
        .filter(|f| f.widget.is_some())
        .collect();
```

Reply:
Taking back this comment, perhaps we won't need this fix if the changes #128 are approved